sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -105,6 +107,15 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO> {
     try {
+      const application =
+        await this.applicationService.processScholasticStanding(
+          locationId,
+          applicationId,
+        );
+      payload.data.applicationOfferingStartDate =
```

Comment:
The API key in form is `applicationOfferingUnformattedStartDate`

![image](https://github.com/user-attachments/assets/8bd29f28-658f-4a43-87c3-3e8626820c5d)

Reply:
Please match the properties with form hidden variables `studyStartDate` `studyEndDate` 
![image](https://github.com/user-attachments/assets/67c074b8-2579-418e-8675-c001dc6bbfb2)